### DISCUSSION TITLE
Add endpoints to InfluxDB/Grafana

### DISCUSSION THREAD

**Discussion by PatRussell87** [Security-Onion-Solutions/securityonion#2589] [Unsupported Versions] [answered]
Is it possible to use the InfluxDB/Grafana install in SecurityOnion for other (non-SO) Linux endpoints in the environment? If so, would any information be needed for the specific SO installed instance to get them added?

Just now playing with Security Onion 2.x and the new tools and I'm really liking it so far. Looking for other opportunities once I have it in production.

Pat

**Comment by TOoSmOotH** [comment, marked as answer]
It would require you opening influxdb up to them on the firewall. Then in your telegraf config on the other hosts you would point to the manager using ssl. You could copy the certs over or disable ssl verification.

**Reply by PatRussell87** [reply]
Thank you! That's what it was looking like but I wasn't sure if I missed something. First time working with this set of tools.
